pom.xml

```diff
@@ -972,15 +972,16 @@
       -->
     <dependencies>
         <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <version>4.8</version>
+            <!--Hamcrest first, then JUnit, then Mockito. See http://goo.gl/e5bJA5-->
```

Comment:
There is a ticket to update the testing tools: mockito, junit, hamcrest. I will tackle that ticket soon.